ktxspec.adoc

```diff
@@ -2237,10 +2213,47 @@ particular version of OpenGL {,ES} or Vulkan.
   https://fgiesen.wordpress.com/2018/12/10/rate-distortion-optimization/[Rate-distortion
   optimization]. The ryg blog, December 18th, 2018.
 
+[appendix#cubemapOrientation]
+== Cubemap Orientation
+
+CAUTION: This appendix is non-normative.
+
+The KTX cubemap coordinate system in <<faceCount>> is directly
+compatible with the Vulkan and OpenGL cube samplers described by the
+face selection tables and equations for calculating (s, t) in
+{url-khr-vulkan}/specs/1.2-extensions/html/vkspec.html#_cube_map_face_selection_and_transformations[section
+15.5.3 Cube Map Face Selection and Transformations] of <<VULKAN12>>
+and
+{url-khr-reg}/OpenGL/specs/gl/glspec46.core.pdf#nameddest=section-8.13[section
+8.13 Cube Map Texture Selection] of <<OPENGL46>>.
+
+<<orientationFigure>> shows graphically how the cubemap images should be
+arranged.
+
+[[orientationFigure]]
+.Cubemap Coordinate System
+[.right.text-center]
+image::cubemap_coord_system.png[]
+
+If using a skybox to render the cubemap the uvw  coordinates passed to the
+cubemap sampler need to match the KTX cubemap coordinate system, that is
+left-handed with +Y up.
+
+If using OpenGL's default clip-space, which is right-handed, you
```

Comment:
OpenGL's default clip-space is left-handed. It's the object space (before eye and projection transforms) that is usually considered right-handed.

Reply:
Fixed.